src/utils/optionsMatches.js

```diff
@@ -0,0 +1,42 @@
+import matchesStringOrRegExp from "./matchesStringOrRegExp"
+
+const extractValueFromInput = (property, input) => {
```

Comment:
In fact, if we remove this special treatment of nodes can we possibly consolidate the two utils into one?